front/app/containers/ProjectsShowPage/timeline/Ideas.tsx

```diff
@@ -47,6 +53,9 @@ const IdeasContainer = ({ projectId, phase, className }: InnerProps) => {
   const config = getMethodConfig(phase.attributes.participation_method, {
     showIdeaFilters: phase.attributes.voting_filtering_enabled,
   });
+  const isIdeasFeedEnabled = useFeatureFlag({ name: 'idea_feed' });
+  const isIdeasFeedPhase = phase.attributes.ideation_method === 'idea_feed';
+  const showIdeasFeedLink = isIdeasFeedEnabled && isIdeasFeedPhase;
```

Comment:
I think it's better not to check the feature flag here, just the participation method. Limiting the feature flag's effect to the ideation_method control in the back office.